### PR TITLE
Moving zeit to root, moving handlers and tests in from airswap-maker-kit

### PR DESCRIPTION
**Pull request by dmosites**

### CODE REVIEW

File: handlers.js [hunk 1]
```diff
@@ -0,0 +1,322 @@
+const ethers = require('ethers')
+const BigNumber = require('bignumber.js')
```
Comment (perich) [inline handlers.js]: bignumber.js needs to be added as a dependency in `package.json`

File: package.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "prettier:run-all": "prettier --write \"src/**/*.{ts,tsx}\"",
     "types": "tsc",
-    "lint": "eslint . --ext .ts,.tsx"
+    "lint": "eslint . --ext .ts,.tsx",
+    "test": "yarn mocha handlers.test.js"
   },
   "license": "Apache-2.0",
   "dependencies": {
```
Comment (perich) [inline package.json]: maker-kit can be removed as a dependency now

File: handlers.js (hunk 1 above)
Comment (dmosites) [reply handlers.js]: Added

File: package.json (hunk 2 above)
Comment (dmosites) [reply package.json]: Removed

File: now.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "maker",
+  "name": "m",
```
Comment (ejwessel) [inline now.json]: is this the name you want?

Comment (dmosites) [reply now.json]: Yes, due to the URL length limitations (bytes32) figure it's best to reduce to a single character.

**Pull request merged**